uint16 lift_position
uint16 tilt_position

uint16 measured_weight